app/components/CreatePasswordForm/containers/CreatePasswordForm.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+
+import Layout from '../components/Layout'
+
+class CreatePasswordForm extends Component {
+  constructor(props) {
```

Comment:
You can replace the constructor by: 
```
state = {
    error: { 
        ...
    }
}
```

And it will work fine as well. Little sugar syntax trick, but do whichever feels you more comfortable.